Instrukcja

```diff
@@ -1,3 +1,4 @@
+                                                           KWARANTANNA 
```

Comment:
Czy to są spacje z przodu? Usuń to!

Reply:
Nie, tabulatory

Reply:
Usuń tabulatory. To nie tak powinno być.
Możesz zostawić 1.